### ISSUE TITLE
bazel-snapshots fails when run with Bazel remote execution

### ISSUE THREAD

**Issue by mortenmj** [cognitedata/bazel-snapshots#80]
```
+ bazel run snaptool --config=remote -- diff --out snapshot.json --format=json --stderr-pretty deployed
WARNING: Option 'javabase' is deprecated
WARNING: Option 'java_toolchain' is deprecated
WARNING: Option 'host_javabase' is deprecated
WARNING: Option 'host_java_toolchain' is deprecated
INFO: Invocation ID: 79704985-d849-4078-8faa-b8f991152287
INFO: Streaming build results to: https://foo.bar/invocation/79704985-d849-4078-8faa-b8f991152287
Loading: 
Loading: 0 packages loaded
Analyzing: target //:snaptool (0 packages loaded, 0 targets configured)
INFO: Analyzed target //:snaptool (0 packages loaded, 3 targets configured).
INFO: Found 1 target...
[0 / 4] [Prepa] BazelWorkspaceStatusAction stable-status.txt
Target //:snaptool up-to-date:
  bazel-bin/snaptool-runner.bash
  bazel-bin/snaptool
INFO: Elapsed time: 1.910s, Critical Path: 0.79s
INFO: 137 processes: 133 remote cache hit, 4 internal.
INFO: Build completed successfully, 137 total actions
INFO: Running command line: bazel-bin/snaptool diff --out snapshot.json '--format=json' --stderr-pretty deployed
INFO: Streaming build results to: https://foo.bar/invocation/79704985-d849-4078-8faa-b8f991152287
INFO: Build completed successfully, 137 total actions
INFO: Build completed successfully, 137 total actions
/bazel-cache/infrastructure/execroot/infrastructure/bazel-out/k8-fastbuild/bin/snaptool: line 16: ../com_cognitedata_bazel_snapshots/snapshots/go/cmd/snapshots/snapshots_/snapshots: No such file or directory
script returned exit code 127
```

SNAPSHOTS on line https://github.com/cognitedata/bazel-snapshots/blob/main/snapshots/runner.tmpl.sh#L6 expands to ../com_cognitedata_bazel_snapshots/snapshots/go/cmd/snapshots/snapshots_/snapshots, and seems to be where the problem originates.

**Comment by mortenmj** [comment]
This problem only occurs if we run with from_source = True. Running from the latest release, and setting from_source = False provides a workaround.

**Comment by mortenmj** [comment]
Closed by #103
